Unix/miapi/ChildList.h

```diff
@@ -163,7 +163,7 @@ MI_INLINE int ChildList_GetCurrentList(_Inout_ ChildList *list, _Out_cap_post_co
         ChildListNode *currentNode = list->headNode;
         (*outboundUsed) = 0;
         while (currentNode && 
-            ((*outboundUsed) < inboundSize))    //redumbdant check to shut prefast up, it does not know list->childCount is how many items in list
+            ((*outboundUsed) < inboundSize))    //redundant check to shut prefast up, it does not know list->childCount is how many items in list
```

Comment:
fwiw, it should be `shut up prefast` (you can split phrases like this) -- but this is a spelling PR, and that's grammar